core/src/avm2/globals/flash/net/URLRequest.as

```diff
@@ -0,0 +1,10 @@
+package flash.net {
+	public final class URLRequest {
+		// NOTE - when implementing properties (e.g. `contentType`, `data`, etc.)
+		// be sure to also check for them in `URLLoader`
+		public var url:String;
```

Comment:
Does Flash actually do any validation of the url in the setter?

Reply:
Not sure! I don't think so, but I wouldn't be surprised if it resets some of the other values.

Naturally this is all insignificant, but eventually we do want these to match the actual definitions of the FP, because this limits the chances for differences to creep in.